core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/AntennaWorkflow.java

```diff
@@ -57,6 +57,12 @@ public AntennaWorkflow(AntennaWorkflowConfiguration antennaWFConfig) {
 
             generatedOutput.putAll(processingState.getAttachables());
 
+            if(!processingState.getFailCausingResults().isEmpty()) {
+                throw new ExecutionException(
+                        makeStringForFailCausingResults(
+                                processingState.getFailCausingResults()));
```

Comment:
A little bit cleaner code would be:
```suggestion
                throw new makeExceptionForFailCausingResults(processingState.getFailCausingResults());
```
with a simple function
```
private ExecutionException makeExceptionForFailCausingResults(Map<String, String> failCausingResults) {
```